src/quo/components/wallet/token_input/view.cljs

```diff
@@ -12,15 +12,17 @@
     [quo.foundations.common :as common]
     [quo.theme :as quo.theme]
     [react-native.core :as rn]
-    [schema.core :as schema]))
+    [schema.core :as schema]
+    [utils.number :as number]))
 
 (defn fiat-format
   [currency num-value conversion]
   (str (get common/currency-label currency) (.toFixed (* num-value conversion) 2)))
 
 (defn crypto-format
   [num-value conversion crypto-decimals token]
-  (str (.toFixed (/ num-value conversion) (or crypto-decimals 2))
+  (str (number/remove-trailing-zeroes
+        (.toFixed (/ num-value conversion) (or crypto-decimals 2)))
```

Comment:
Numbers can be of 0.000100000, so need to remove trailing zeroes

Reply:
a test maybe?